tests/snappi_tests/multidut/pfcwd/files/pfcwd_multidut_runtime_traffic_helper.py

```diff
@@ -0,0 +1,255 @@
+import time
+import logging
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.snappi_tests.snappi_helpers import get_dut_port_id          # noqa: F401
+from tests.common.snappi_tests.common_helpers import start_pfcwd, stop_pfcwd
+from tests.common.snappi_tests.port import select_ports, select_tx_port       # noqa: F401
+from tests.common.snappi_tests.snappi_helpers import wait_for_arp
+from tests.common.snappi_tests.snappi_test_params import SnappiTestParams
+
+DATA_FLOW_NAME = "Data Flow"
+DATA_PKT_SIZE = 1024
+DATA_FLOW_DURATION_SEC = 15
+PFCWD_START_DELAY_SEC = 3
+SNAPPI_POLL_DELAY_SEC = 2
+TOLERANCE_THRESHOLD = 0.05
+
+logger = logging.getLogger(__name__)
+
```

Comment:
Please add comments about what does this function do and how ?

Reply:
this is a generic logger function used in almost all sonic-mgmt cases for create logger object for log print out, don't think a comment is required for this